<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pageAppearance</class>
 <widget class="QWidget" name="pageAppearance">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>9</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox1">
         <property name="title">
          <string>Theme</string>
         </property>
         <layout class="QGridLayout" name="groupBox1_gridLayout" columnstretch="0,0,1">
          <property name="horizontalSpacing">
           <number>16</number>
          </property>
          <property name="verticalSpacing">
           <number>8</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Labwc theme</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="openboxTheme"/>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_1">
            <property name="text">
             <string>Icon theme</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="iconTheme"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox2">
         <property name="title">
          <string>Window Drop Shadows</string>
         </property>
         <layout class="QGridLayout" name="groupBox2_gridLayout" columnstretch="0,0,1">
          <property name="horizontalSpacing">
           <number>16</number>
          </property>
          <property name="verticalSpacing">
           <number>8</number>
          </property>
          <item row="0" column="0">
           <widget class="QCheckBox" name="dropShadows">
            <property name="text">
             <string>Enable shadows</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="dropShadowsOnTiled">
            <property name="text">
             <string>Enable on tiled windows</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <spacer name="horizontalSpacer_11">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox3">
         <property name="title">
          <string>Titlebar</string>
         </property>
         <layout class="QGridLayout" name="groupBox3_gridLayout" columnstretch="0,0,1">
          <property name="horizontalSpacing">
           <number>16</number>
          </property>
          <property name="verticalSpacing">
           <number>8</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Corner radius</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="cornerRadius">
            <property name="maximum">
             <number>24</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Maximized decoration</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="maximizedDecoration"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox4">
         <property name="title">
          <string>Advanced</string>
         </property>
         <layout class="QGridLayout" name="groupBox4_gridLayout" columnstretch="0,0,1">
          <property name="horizontalSpacing">
           <number>16</number>
          </property>
          <property name="verticalSpacing">
           <number>8</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Decoration mode</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="decoration"/>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
